komodo-relational/src/main/java/org/komodo/relational/ViewDdlBuilder.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * JBoss, Home of Professional Open Source.
+*
+* See the LEGAL.txt file distributed with this work for information regarding copyright ownership and licensing.
+*
+* See the AUTHORS.txt file distributed with this work for a full listing of individual contributors.
+*/
+package org.komodo.relational;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.komodo.relational.model.Column;
+import org.komodo.relational.model.Table;
+import org.komodo.spi.KException;
+import org.komodo.spi.constants.StringConstants;
+import org.komodo.spi.lexicon.TeiidSqlConstants;
+import org.komodo.spi.repository.Repository.UnitOfWork;
+import org.komodo.spi.runtime.version.TeiidVersion;
+import org.komodo.spi.runtime.version.TeiidVersionProvider;
+
+/**
+ * ViewDdlBuilder
+ * methods for generating view DDL for various scenarios
+ */
+public class ViewDdlBuilder {
```

Comment:
Something seems to have gone a little awry with the space formatting in this class? Some tabs seem far large than the usual 4 spaces while other are smaller than 4 spaces and some methods are not tab-indented at all. Is it maybe due to a new eclipse install that doesn't have formatting switched on?